src/main/scala/net/ruippeixotog/scalascraper/browser/Browser.scala

```diff
@@ -15,6 +15,16 @@ trait Browser {
   def parseFile(path: String, charset: String): Document = parseFile(new File(path), charset)
   def parseFile(path: String): Document = parseFile(new File(path), "UTF-8")
 
+  def parseResource(name: String, charset: String = "UTF-8"): Document = {
+    val stream = getClass.getResourceAsStream(name)
+    val document = parseInputStream(stream, charset)
+    stream.close()
+    document
+  }
+
+  /** Does not close the InputStream after reading */
+  def parseInputStream(inputStream: InputStream, charset: String = "UTF-8"): Document
```

Comment:
I didn't have any preference either way. Jsoup doesn't seem to close the `InputStream`, and generally "whoever creates a stream is responsible for closing it" makes sense in Java.

Come think of it, though, in Scala this makes less sense, and closing the `InputStream` inside scala-scraper might move some complexity from the consumer to the library - which is probably a good thing.